Release Notes/Release Notes v2.10.0.md

```diff
@@ -1,10 +1,14 @@
-# .NET Driver Version 2.10.0 Placeholder Release Notes
+# .NET Driver Version 2.10.0 Release Notes
 
-The main new features in 2.10.0 are:
+The main changes in 2.10.0 are:
+
+1. A number of minor bug fixes
+2. New ReplaceOptions parameter for the ReplaceOne CRUD methods
+3. Client-side field level encryption (FLE)
```

Comment:
Can we add a link to the a documentation page?  Either to client-side-encryption.md (or whatever it's called) or to https://docs.mongodb.com/manual/core/security-client-side-encryption/

Reply:
I'll try and figure out what such a link would look like.